contrib/asff.tpl

```diff
@@ -45,7 +45,7 @@
         "Remediation": {
             "Recommendation": {
                 "Text": "More information on this vulnerability is provided in the hyperlink",
-                "Url": "{{ index .References 0 }}"
+                "Url": "https://avd.aquasec.com/nvd/{{ .VulnerabilityID | toLower }}"
```

Comment:
What's going to happen if `VulnerabilityID` is not CVE-ID?

Reply:
Is that a possibility? If so what could it be? I was assuming it never would be. But in that case today it will just be a 404.

Reply:
There are many vulnerability IDs other than CVE-ID. Most of the vulnerabilities for programming language library are not CVE-ID assigned.  Do you think 404 is acceptable?

Reply:
oh!!! I see. Yeah that's not acceptable to be left in like that. I have added a filter to remove such links here https://github.com/aquasecurity/trivy/pull/709/commits/3666046f4710208394bcab49a7d12e129012f7ae

And some new test cases that exercise these paths:
https://github.com/aquasecurity/trivy/pull/709/commits/3666046f4710208394bcab49a7d12e129012f7ae#diff-358c0e4a4c61cd63c630c1f0eb03bcb31c4e5e26a1c8400ff4fffc05454ed259R333-R342
and 
https://github.com/aquasecurity/trivy/pull/709/commits/3666046f4710208394bcab49a7d12e129012f7ae#diff-358c0e4a4c61cd63c630c1f0eb03bcb31c4e5e26a1c8400ff4fffc05454ed259R99-R108